src/lib.rs

```diff
@@ -18,16 +22,14 @@
 //! ```
 
 mod render;
+pub mod slug;
```

Comment:
This crate's public API is not too big so I think it's fine to just export the `Slugify` trait in the crate root and not make this module public.

Reply:
Sure ... done.